libs/ngrx-traits/signals/src/lib/with-calls/with-calls.ts

```diff
@@ -255,7 +271,7 @@ export function withCalls<
                             call.onSuccess &&
                             call.onSuccess(result, params);
                         }),
-                        first(),
+                        takeUntilDestroyed(destroyRef),
```

Comment:
the destroy ref is optional, you dont need to pass it it will automatically grab the current one

Reply:
Oh thanks for letting me know, I never realised that.